twiggy/tests/tests.rs

```diff
@@ -59,17 +61,29 @@ macro_rules! test {
                     cmd.push(' ');
                     cmd.push_str($args);
                 )*
-                println!("\n`{}` did not have the expected output!\n", cmd);
-
+                println!(
+                    "\n{} {}\n",
+                    format!("`{}`", cmd).red(),
+                    "did not have the expected output!".red()
+                );
                 println!("--- {}", expected_path);
-                println!("+++ actually generated by `{}`", cmd);
+                println!(
+                    "{} {}\n",
+                    "+++ actually generated by".red(),
+                    format!("`{}`", cmd).red()
+                );
                 for diff in diff::slice(&expected_lines, &actual_lines) {
                     match diff {
-                        diff::Result::Left(l) => println!("-{}", l),
+                        diff::Result::Left(l) => {
+                            println!("{}", format!("-{}", l).red())
+                        }
                         diff::Result::Both(l, _) => println!(" {}", l),
-                        diff::Result::Right(r) => println!("+{}", r),
+                        diff::Result::Right(r) => {
+                            println!("{}", format!("+{}", r).red())
```

Comment:
Might as well match normal colorized diff output and use `.green()` for added lines

Reply:
@fitzgen: I also thought about `.green()` for the `+` lines, but then I thought: No! `.green()` looks to good for something so wrong ;P 

And those all-red diffs in failing tests are actually quite common in testing frameworks... like [pytest](https://pytest.org)

Reply:
Ok!